services/appflowy-collaborate/src/collab/disk_cache.rs

```diff
@@ -54,6 +56,17 @@ impl CollabDiskCache {
     transaction: &mut Transaction<'_, sqlx::Postgres>,
   ) -> AppResult<()> {
     insert_into_af_collab(transaction, uid, workspace_id, params).await?;
+    if let Some(em) = &params.embeddings {
```

Comment:
Is this intend to log these inforamtion? it might be lots of logs

Reply:
Tbh. I don't think it will be that much. Are we keeping the metrics on how often collabs are inserted into database?

Reply:
No. we don't implement the metrics yet. Will see how it going